jpa/boot-data-envers/src/test/java/com/example/envers/web/controllers/CustomerControllerIT.java

```diff
@@ -88,6 +88,44 @@ void shouldFindCustomerRevisionsById() throws Exception {
                     .andExpect(jsonPath("$[0].revisionNumber", notNullValue()))
                     .andExpect(jsonPath("$[0].revisionType", is("INSERT")));
         }
+
+        @Test
+        void shouldFindCustomerHistoryById() throws Exception {
+            Customer customer = customerList.getFirst();
+            customerRepository.saveAndFlush(customer.setAddress("newAddress"));
+            Long customerId = customer.getId();
+
+            mockMvc.perform(get("/api/customers/{id}/history?page=0&size=10&sort=revision_Number,desc", customerId))
+                    .andExpect(status().isOk())
+                    .andExpect(jsonPath("$.data.size()", is(2)))
+                    .andExpect(jsonPath("$.totalElements", is(2)))
+                    .andExpect(jsonPath("$.pageNumber", is(1)))
+                    .andExpect(jsonPath("$.totalPages", is(1)))
+                    .andExpect(jsonPath("$.isFirst", is(true)))
+                    .andExpect(jsonPath("$.isLast", is(true)))
+                    .andExpect(jsonPath("$.hasNext", is(false)))
+                    .andExpect(jsonPath("$.hasPrevious", is(false)))
+                    .andExpect(jsonPath("$.data[0].entity.id", is(customer.getId()), Long.class))
+                    .andExpect(jsonPath("$.data[0].entity.name", is(customer.getName())))
+                    .andExpect(jsonPath("$.data[0].entity.address", is(customer.getAddress())))
+                    .andExpect(jsonPath("$.data[0].revisionNumber", notNullValue()))
+                    .andExpect(jsonPath("$.data[0].revisionType", is("UPDATE")))
+                    .andExpect(jsonPath("$.data[0].revisionInstant", notNullValue()));
+        }
+
+        @Test
+        void cantFindCustomerHistoryById() throws Exception {
```

Comment:
@coderabbitai, convert this code to use RestAssured instead of mock mvc

Reply:
@coderabbitai, you have forgot to update RestAssured.port , please update the solution

Reply:
@coderabbitai, you can replace int port with below code

```java
    @LocalServerPort
    private int localServerPort;
```